protocol/delta-sharing-protocol-api.yml

```diff
@@ -670,7 +676,7 @@ components:
           items:
             type: string
         jsonPredicateHints:
```

Comment:
can you factor out this object into a separate schema and then remove the commented one?